app/src/main/java/net/gsantner/markor/format/markdown/MarkdownHighlighterPattern.java

```diff
@@ -18,7 +18,7 @@
     HEADING_SIMPLE(Pattern.compile("(?m)^(#{1,6}\\s.*$)")),
     LINK(Pattern.compile("\\[([^\\[]+)\\]\\(([^\\)]+)\\)")),
     LIST_UNORDERED(Pattern.compile("(\\n|^)\\s{0,16}([*+-])( \\[[ xX]\\])?(?= )")),
-    LIST_ORDERED(Pattern.compile("(?m)^\\s{0,16}\\d+\\.\\s")),
+    LIST_ORDERED(Pattern.compile("(?m)^\\s{0,16}(\\d+)(:?\\.|\\))\\s")),
```

Comment:
Handle ordered lists demarcated with ')' per commonmarkd

Reply:
If you think it improves something it's ok.